### DISCUSSION TITLE
next/font declarations support

### DISCUSSION THREAD

**Discussion by MauricioRobayo** [storybookjs/storybook#24979] [Help] [answered]
### Summary

When is support for `next/font` `declarations` coming?

### Additional information

I see on the [@storybook/next](https://www.npmjs.com/package/@storybook/nextjs#not-supported-features-of-nextfont) documentation that [`declarations`](https://nextjs.org/docs/pages/api-reference/components/font#declarations) is not **yet** supported.

Wanted to know if there is an ETA for supporting this.

I'm using [descent-override](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/descent-override) for a font and because of the missing support components don't look as expected on Storybook..

**Comment by valentinpalkovic** [comment]
Would you like to contribute? Shouldn't be too hard to implement. I can also help if questions arise.

**Reply by MauricioRobayo** [reply]
I would love to contribute @valentinpalkovic! I have never looked into Storbybook codebase before but will love to dig around.

**Reply by MauricioRobayo** [reply, marked as answer]
here is a ~~draft~~ PR: https://github.com/storybookjs/storybook/pull/24983

**Reply by joevaugh4n** [reply]
Amazing! Thanks for spinning this up, @MauricioRobayo!
